src/Discord.Net/Entities/Emotes/IGuildEmote.cs

```diff
@@ -1,7 +1,44 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
 namespace Discord
 {
-    public interface IGuildEmote : IEmote
+    public interface IGuildEmote : IEmote, ISnowflakeEntity, IDeletable
     {
-        // TODO
+        /// <summary>
+        ///     Gets whether this emoji is managed by an integration.
+        /// </summary>
+        /// <returns>
+        ///     A boolean that determines whether or not this emote is managed by a Twitch integration.
```

Comment:
Minor nit, but this doc probably shouldn't exclusively mention Twitch. I'm sure other integrations that can create managed emoji will come in the future, most likely YouTube being ntext.